ferrocene/ferrocene
# Update "Upstream pulls" in "Internal Procedures"

This PR updates the existing procedures with current procedures.

To test:
 `x doc ferrocene/doc/internal-procedures/ --serve` 

In review:
Please check if this makes sense, or if there is something recurrent that I forgot.

Questions to reviewers:
What more to say on ignored tests? When is it good practice to ignore a test?
Do we need to describe `run completions`?
Is it `x fmt --check` for the licensing?
Shall I add structured handover?

Added @Hoverbear points:
- monitor the cricleci job and cancel eagerly on first failure.
- always try running locally with x check or x build
